packages/wallets/src/features/cashier/flows/WalletResetBalance/WalletResetBalance.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { useActiveWalletAccount, useRequest } from '@deriv/api';
```

Comment:
@wojciech-deriv please, use useMutation instead of useRequest, it's deprecated now

Reply:
hey, updated :)